std_msgs/Header header
int8 aircraftStatus
int8 fcuMode
int8 fcuStatus
int8 landDetectorState